app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class PostsController < ApplicationController
+  before_action :signed_in_user, only: %i[new create]
+
+  def index
+    @posts = Post.all
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = current_user.posts.build(post_params)
+    @post.user_id = current_user.id
+    @post.save
+    redirect_to root_path
+  end
+
+
+
+
+
+
+  private
+
+  def post_params
+    parame.require(:post).permit(:title, :body)
```

Comment:
am unable to create a post due to this typo..... `parame`

Reply:
At first, we had a branch which we could not fix, so we used the branch we practised on , instead of starting from scratch.